bssl-compat/CMakeLists.txt

```
@@ -54,7 +55,8 @@ add_library(bssl-compat
     )
 
 add_dependencies(bssl-compat
-        OpenSSL)
+        OpenSSL::SSL
+        OpenSSL::Crypto)
 
 target_link_libraries(bssl-compat PRIVATE OpenSSL::Crypto)
```

```suggestion
target_link_libraries(bssl-compat INTERFACE OpenSSL::Crypto)
```
To get the `test_rand` binary to link I had to change this, so that linking with `bssl-compat` also adds `libcrypto` to the link line.

---

Thanks, added fix!